--- openmw-0.9.0-source/CMakeLists.txt~	2011-02-09 22:43:00.000000000 +0100
+++ openmw-0.9.0-source/CMakeLists.txt	2011-03-07 16:51:17.707658912 +0100
@@ -267,10 +267,10 @@
 endif (APPLE)
 
 # Compiler settings
-if (CMAKE_COMPILER_IS_GNUCC)
-    #add_definitions (-Wall -Werror)
-    add_definitions (-Wall)
-endif (CMAKE_COMPILER_IS_GNUCC)
+#if (CMAKE_COMPILER_IS_GNUCC)
+#    #add_definitions (-Wall -Werror)
+#    add_definitions (-Wall)
+#endif (CMAKE_COMPILER_IS_GNUCC)
 
 # Apple bundling
 if (APPLE)
